src/rpc/blockchain.cpp

```diff
@@ -1283,6 +1283,10 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "  \"chainwork\": \"xxxx\"     (string) total amount of work in active chain, in hexadecimal\n"
             "  \"pruned\": xx,             (boolean) if the blocks are subject to pruning\n"
             "  \"pruneheight\": xxxxxx,    (numeric) lowest-height complete block stored\n"
+            "  \"initialblockdownload\":   (boolean) Is in initial block download\n"
+            "  \"datareceived\": xxxxxx,   (numeric) Data recieved in bytes\n"
+            "  \"datasent\": xxxxxx,       (numeric) Data sent in bytes\n"
+            "  \"datadir\": xxxxxx,        (string) Current data directory\n"
```

Comment:
I'm not sure if exposing local disk folder structure in rpc is a good idea tbh.

Reply:
Could do only if wallet is unlocked?

Reply:
No, I mean, exposing local folder structure in rpc could potentially leak some sensitive information about filesystem to someone who should have no access to it and who should only have access to blockchain stats for example.